openrr-apps/src/bin/robot_command.rs

```diff
@@ -128,4 +128,48 @@ mod tests {
     fn assert_app() {
         RobotCommandArgs::command().debug_assert();
     }
+
+    #[test]
+    fn assert_sample_command() {
+        let bin = env!("CARGO_BIN_NAME");
+        let sample_command = RobotCommandArgs::try_parse_from(&[
+            bin,
+            "--config-path=./openrr-apps/config/sample_robot_client_config_for_urdf_viz.toml",
+            "load_commands",
+            "./openrr-apps/command/sample_cmd_urdf_viz.txt",
```

Comment:
This does not seem robust as it depends on the current directory.

Instead, I suggest using [the `CARGO_MANIFEST_DIR` environment variable](https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-crates) to get the absolute path to the `openrr-apps` directory. For example:

```rs
let manifest_dir = Path::new(env!("CARGO_MANIFEST_DIR"));
let sample_command_path = manifest_dir.join("command/sample_cmd_urdf_viz.txt");
```

Reply:
Thank you for your review. I updated code.